footer.php

```diff
@@ -1,13 +1,28 @@
 			<!-- footer -->
 			<footer class="footer" role="contentinfo">
-
-				<!-- copyright -->
-				<p class="copyright">
-					&copy; <?php echo date('Y'); ?> Copyright <?php bloginfo('name'); ?>. <?php _e('Powered by', 'html5blank'); ?>
-					<a href="//wordpress.org" title="WordPress">WordPress</a> &amp; <a href="//html5blank.com" title="HTML5 Blank">HTML5 Blank</a>.
-				</p>
-				<!-- /copyright -->
-
+			
+				<div class="container-fluid footer-align">
+					<div class="row icon-width">
+						<a href="#"><i class="fab fa-facebook fa-2x social"></i></a>
+					</div>
+					<div class="row footer-info">
+						<div class="center">
+								<h1>We Hear You Foundation</h1>
```

Comment:
We need to make the footer content dynamic so they can change it from wp-admin. Once we get the merge together, I can tackle that on another branch.